runtime-sdk/modules/contracts/src/lib.rs

```diff
@@ -646,6 +646,17 @@ impl<Cfg: Config> Module<Cfg> {
         Self::load_code_info(ctx, args.id)
     }
 
+    #[handler(query = "contracts.CodeStorage")]
+    pub fn query_code_storage<C: Context>(
```

Comment:
Add unit test (in `test.rs`) for this added query (see where the existing tests are testing queries).